packages/website/components/account/paymentTable.js/paymentTable.js

```diff
@@ -32,30 +50,36 @@ const PaymentTable = ({ plans, currentPlan, setPlanSelection, setIsPaymentPlanMo
             </div>
             {plans.map(plan => (
               <div
-                key={plan.title}
-                className={`billing-card card-transparent ${currentPlan?.id === plan.id ? 'current' : ''}`}
+                key={plan.metadata['UI Label']}
```

Comment:
since metadata is loosely typed, this `plan.metadata['UI Label']` can be undefined, and then weird things may start happening in the loop with many `key={undefined}`

Reply:
updated (see [here](https://github.com/web3-storage/web3.storage/pull/2152#discussion_r1065217803))